projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -6823,7 +6823,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         let res = !parentElement ||
             parentElement.clientHeight === 0 ||
             parentElement.clientHeight === renderedHeight;
-        if ((!this.platform.isChromium && !this.platform.isFirefox) || this._autoSize) {
```

Comment:
While I love the removal of browser checks, the original fix from #12033 doesn't make any browser-specific mention at all, nor do the related issues as far as I can see. Kind of curios now as to why that was and wondering if that's been cleared out.

Reply:
I couldn't find any browser specific issue mentioned there either and it should be fine to check if the content stretches the container in any browser IMO. Perhaps @ddincheva might remember. Adding her as a reviewer as well.